pom.xml

```diff
@@ -467,7 +467,7 @@
           <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>7.8</version>
+            <version>7.9-SNAPSHOT</version>
```

Comment:
>Why did we use stable version but not current SNAPSHOT version before?

Maven release plugin is failing if any snapshot version is found, as release version have to be stable and based on strict versions